migration/20170713-17-move-externalintegration-columns-to-settings.sql

```diff
@@ -37,3 +37,10 @@ UPDATE collections c
 SET external_integration_id = e.id
 FROM externalintegrations e
 WHERE e.protocol = 'OPDS Import' and c.external_integration_id is null;
+
+ALTER TABLE collections DROP COLUMN protocol;
```

Comment:
It should always be there, but it makes sense to add `if exists` in general. No sense creating unhelpful migration errors.